plugins/eos/gs-plugin-eos.c

```diff
@@ -822,6 +827,19 @@ app_is_banned_for_personality (GsPlugin *plugin, GsApp *app)
 	         g_str_has_prefix (id, "com.endlessm.encyclopedia")));
 }
 
+static gboolean
+app_is_banned_coding_app (GsPlugin *plugin, GsApp *app)
+{
+	GsPluginData *priv = gs_plugin_get_data (plugin);
+	const char *id = gs_app_get_id (app);
```

Comment:
It's make the code that calls it extra long, so I guess that's why it's written like that...

Reply:
I don't see the problem with that, but perhaps it's my WebKit past that keeps "hunting" me...

Reply:
Can't see any change in the new PR, I suppose you made a decision here :-), although I'm a bit puzzled on why you consider less readable caching the result of a function that will be reused in every iteration of a loop but, however, are happy with declaring a variable that's only used once just because it would make a line extra long? :-)

Reply:
This was not my code, that's why I said it's probably why the var has been declared like that.
But I can change it...